idseq_pipeline/commands/common.py

```diff
@@ -20,6 +20,7 @@
 ACCESSION2TAXID = ("%s/%s/accession2taxid.db.gz" % (base_s3, base_dt))
 base_s3 = bucket + "/taxonomy"
 LINEAGE_SHELF = ("%s/%s/taxid-lineages.db" % (base_s3, base_dt))
+INVALID_CALL_BASE_ID = -1e8 # don't run into -2e9 limit. current largest taxid is around 2e6 so should be fine
```

Comment:
this -2e9 limit is only when using 32-bit integers yes?   if using 32-bit floating point numbers, the limit is even less (there are only 23 bits of mantissa in fp32... so you start losing precision a lot sooner)

i recommend not using "1e8" notation if your domain is the integers, since "-1e8" is a floating point number;   you can say instead `-(10 ** 8)`

and if you ensure that SQL and rails are both using 64-bit integers for this field, then `-(10*9)` is just fine

Reply:
Thanks. It's the limit for the signed integer column in the idseq-web mysql database